src/fontra/views/editor/editor.js

```diff
@@ -128,8 +129,7 @@ export class EditorController {
       async (...args) => await this.editListenerCallback(...args)
     );
 
-    this.clipboardFormatController = new ObservableController({ format: "glif" });
-    this.clipboardFormatController.synchronizeWithLocalStorage("fontra-clipboard-");
+    this.clipboardFormatController = clipboardFormatController;
```

Comment:
Suggest to just use `clipboardFormatController` instead of `this.clipboardFormatController`.

Reply:
But will the change not break other parts?

Reply:
> But will the change not break other parts?

You can do a global search for `.clipboardFormatController`. It looks like it's used by the gear panel and editor.js, and nothing else.

Reply:
ok, done.